src/blueapi/config.py

```diff
@@ -9,7 +10,18 @@
 LogLevel = Literal["NOTSET", "DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"]
 
 
-class StompConfig(BlueapiBaseModel):
+class SourceKind(str, Enum):
+    planFunctions = "planFunctions"
+    deviceFunctions = "deviceFunctions"
+    dodal = "dodal"
```

Comment:
Very minor, but enums count as constants under PEP 8 so names should in SCREAMING_SNAKE_CASE

```suggestion
    PLAN_FUNCTIONS = "planFunctions"
    DEVICE_FUNCTIONS = "deviceFunctions"
    DODAL = "dodal"
```